envs/mamba-env.yaml

```diff
@@ -4,6 +4,7 @@ packages:
   - feedstock : https://github.com/conda-forge/pybind11-feedstock
     git_tag : 3d3cbe7e8422e78a1c315004ed14480e00f30b59
     patches :
+           - feedstock-patches/pybind11/pybind11-abi_build.patch
```

Comment:
Do we need both these patches?

Reply:
yes. As in remote feedstock we are removing build skip part for pybind11-abi

Reply:
The change has now become redundant, lets have a single patch with your change added to it.